playwright-e2e/dss/pages/page-base.ts

```diff
@@ -416,9 +415,10 @@ export default abstract class PageBase implements PageInterface {
       : new Question(this.page, { prompt: 'Signature' }).toInput();
   }
 
-  async getDate(): Promise<Date> {
+  async getDisplayedDate(): Promise<string> {
```

Comment:
to avoid confusion around same-named function with `date-utils.ts` `getDate` function.